### DISCUSSION TITLE
Some Sugesstions

### DISCUSSION THREAD

**Discussion by xalamenders** [Swordfish90/Lemuroid#546] [Ideas]
I just found this emulator while looking for genesis emulator for my phone earlier this week and already finished phantasy star 4, IMO this is one of the best emulator out there. Its like retroarch without all of its fuzz, some suggestion tho..

1. More configureable control. While i like the neatness of current control layout, please add function to place each button group.

2. Cheat support, well... Like it or not, this one is necessary to be present in emulator.

That's all for now, keep up the good works dev!

**Comment by SethPeters91** [comment]
I agree. The lack of touch controller configuration (aside from the size and positioning) makes it difficult to make the most out of your screen space. Playing 3DS especially is really difficult to see due to not only the lack of touch control configuration but also the lack of screen configuration as well. I would use Citra but the touch controls are awful and unresponsive

**Comment by DuytheD** [comment]
what u think about adding camera or QR scan?

**Reply by kangurek-kao** [reply]
Camera support would be great for Gambatte, PPSSPP and Citra cores
